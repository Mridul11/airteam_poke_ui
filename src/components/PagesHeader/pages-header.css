.center-header {
    display: flex;
    justify-content: center;
    align-items: center;
    margin: auto;
}