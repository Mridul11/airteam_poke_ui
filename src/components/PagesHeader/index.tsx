import './pages-header.css';
import { PageHeader } from 'antd';

export default function PagesHeader() {
  return (
    <div className='site-page-header-ghost-wrapper center-header'>
      <PageHeader
        ghost={false}
        title='POKEMONDEK'
        subTitle='App to show pokemon and their power'
      ></PageHeader>
    </div>
  );
}
