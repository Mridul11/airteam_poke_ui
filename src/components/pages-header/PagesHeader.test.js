import { act, render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import PagesHeader from '.';
import AddWatchMedia from '../../utils/test-utils/add-matchmedia';

describe('PagesHeader component', () => {
  beforeAll(AddWatchMedia);

  test('renders PagesHeader element', async () => {
    await act(async () =>
      render(
        <MemoryRouter>
          <PagesHeader />
        </MemoryRouter>
      )
    );
    const PagesHeaderElement = screen.getByText('POKEDEK');
    expect(PagesHeaderElement).toMatchSnapshot();
  });
});
