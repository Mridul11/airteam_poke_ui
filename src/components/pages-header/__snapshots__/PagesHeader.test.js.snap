// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesHeader component renders PagesHeader element 1`] = `
<span
  class="ant-menu-title-content"
>
  POKEDEK
</span>
`;
